caffe2/core/storage.h

```diff
@@ -43,14 +75,22 @@ class StorageImpl {
     return data_type_.Match<T>();
   }
 
-  const void* data_ptr() const {
+  const void* data() const {
     return data_ptr_.get();
   }
 
-  void* data_ptr() {
+  void* data() {
     return data_ptr_.get();
   }
 
+  DataPtr& data_ptr() {
+    return data_ptr_;
+  }
+
+  const DataPtr& data_ptr() const {
```

Comment:
Thanks, convergences like this are much appreciated. And please feel free to push back if you think the API should look a little different; we were a lot less strict about API of Storage C++ side.